package-lock.json

```diff
@@ -8772,7 +8772,8 @@
     "i18n-js": {
       "version": "3.7.1",
       "resolved": "https://registry.npmjs.org/i18n-js/-/i18n-js-3.7.1.tgz",
-      "integrity": "sha512-xrRzCeda5ZC0u0yRN+dMdidtda0N+f7t7Pek0ajWb+iyKqSGdrMmuBtbNpWJWY5N4Th0cxbp/BR57zSPdrM3Rw=="
+      "integrity": "sha512-xrRzCeda5ZC0u0yRN+dMdidtda0N+f7t7Pek0ajWb+iyKqSGdrMmuBtbNpWJWY5N4Th0cxbp/BR57zSPdrM3Rw==",
+      "dev": true
```

Comment:
Are you referring to the `dev` ? I saw this change in the current master branch. I think I got this change when I merge with master.. not sure why it still flag as a diff here since master already have this change

Reply:
Yup yup! Ohhh, okok, hmm 🤔  okok, lets leave it first then